options.mk

```diff
@@ -91,3 +91,8 @@ credentials+discovery+ecommerce+insights+lms+registrar+cms
 # Separated by plus signs. Listed in alphabetical order for clarity.
 THIRD_PARTY_SERVICES ?= \
 chrome+coursegraph+elasticsearch710+firefox+memcached+mongo+mysql57+mysql80+opensearch12+redis+namenode+datanode+resourcemanager+nodemanager+sparkmaster+sparkworker+vertica
+
+# Set default theme for MFEs
+ifndef PARAGON_BRAND_PACKAGE
+  PARAGON_BRAND_PACKAGE = edx/brand-edx.org
```

Comment:
[curious] Should the `PARAGON_BRAND_PACKAGE` be defined as the full NPM module's name? It seems like the `@` is concatenated within `microfrontend.yml` vs. included in the `PARAGON_BRAND_PACKAGE` definition.

Any reason this couldn't be `@edx/brand-edx.org` here, to representing the full NPM module name vs. splitting it across multiple files? For example, what if we ended up using a brand package that wasn't associated with an organization, i.e. not prefixed with `@` (unlikely but demonstrates the question)?